### DISCUSSION TITLE
Syringe hint

### DISCUSSION THREAD

**Discussion by u1ps** [etlegacy/etlegacy#1922] [Ideas]
What about adding some kind of notice for medic whenever someone can be healed to full hp with syringe. It can sometimes get confusing to guess if players HP is low enough for syringe heal. 

Here is just one idea: syringe icon next to HP bar when Health is that low.
![med](https://user-images.githubusercontent.com/99591943/156910289-f7ea4cb5-d059-46af-a868-c224733529e4.png)

**Comment by mittermichal** [comment]
I like the idea, but it might be pretty hard to implement because syringe healing is done in lua, not direct part of mod

**Reply by u1ps** [reply]
Maybe does not have to be through lua? There must be some part that controls the health bar UI as it changes colors it does track some HP ranges. Not familiar how it works exactly but maybe if the value is known UI would just have to display the icon. Similar thing already works when a person dies, medic icon. Similar code could be used to get the icon show at certain value next to bar?

**Reply by mittermichal** [reply]
the syringe healing (not the regular reviving) is done via lua, so this would have to be too

**Reply by u1ps** [reply]
i mean maybe it is possible to control just the icon through modification of the Health bar code? Leave the Lua syringe part as it is. Whatever the code is behind Health bar, it would just have to display  icon with offset value at below certain HP.

**Reply by ensiform** [reply]
The health bar code is local changes based on delta between previous and current snapshot received, I don't think the health bar is anything more than red herring here.
